provision.py

```diff
@@ -38,6 +36,7 @@
         "yui-compressor",
         "puppet",               # Used by lint-all
         "gettext",              # Used by makemessages i18n
+        "libsasl2-dev",
```

Comment:
what is this needed for?  Building django-redis?  Regardless, can you add a comment for why it's needed?